modules/virt-preparing-container-disk-for-vms.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+[id="virt-preparing-container-disk-for-vms_{context}"]
+= Preparing a container disk for virtual machines
+
+A container disk must be built with a virtual machine image and pushed
+to a registry before it can be attached to a virtual machine as a containerDisk volume.
+
+.Prerequisites
+
+* You have root or sudo privileges on your local system to install podman and change the owner
```

Comment:
Only to install podman.

The uid is changed inside the Dockerfile, no need for root on the host